cmd/xl-v1-utils.go

```diff
@@ -297,9 +293,6 @@ func readXLMeta(ctx context.Context, disk StorageAPI, bucket string, object stri
 	// Reads entire `xl.json`.
 	xlMetaBuf, err := disk.ReadAll(bucket, path.Join(object, xlMetaJSONFile))
 	if err != nil {
-		if err != errFileNotFound {
-			logger.LogIf(ctx, err)
```

Comment:
Can't we log this like, once in a while ? it is good to see problems related to access to other nodes

Reply:
I can change this to call `LogOnceIf`